src/modules/Footer/Footer.tsx

```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components";
```

Comment:
If you don't use it / it's not ready / it's not part of the task - please remove this file / don't send it to PR